src/transformers/modeling_utils.py

```diff
@@ -1266,6 +1270,12 @@ def load(module: nn.Module, prefix=""):
             if len(error_msgs) > 0:
                 error_msg = "\n\t".join(error_msgs)
                 raise RuntimeError(f"Error(s) in loading state_dict for {model.__class__.__name__}:\n\t{error_msg}")
+
+        # tie unintialized modules
+        unintialized_modules = model.retrieve_modules_from_weights(missing_keys)
```

Comment:
```suggestion
        unintialized_modules = model.retrieve_modules_from_names(missing_keys)
```
The `missing_keys` are weight names, not actual weights.

Reply:
neither is very intuitive, perhaps `from_key_names` or `from_param_names`?

and `unintialized_modules` is missing an `i` :) would be easier for you to rename in an editor.